src/ode_default_alg.jl

```diff
@@ -56,11 +54,23 @@ function default_algorithm(prob::DiffEqBase.AbstractODEProblem{uType,tType,inpla
     else # :auto ∈ alg_hints
       if (!(uEltype <: Float64) && !(uEltype <: Float32)) || tol_level == :extreme_tol
         # Most likely higher precision, so use a higher order method
-        alg = AutoVern9(Rodas5(autodiff=false),lazy=!callbacks)
+        if length(prob.u0) > 100
+            alg = AutoVern9(KenCarp47(autodiff=false),lazy=!callbacks)
```

Comment:
Any reason for this to not use QNDF?

Reply:
Does autoswitch work with multistep? I don't think we have tested that exact combo. I am afraid it might have weird interactions with the history cache, so this might be safer.

Reply:
Ah, right. QNDF doesn't play well with restarts.